staging/src/k8s.io/apiserver/pkg/util/proxy/streamtranslator.go

```diff
@@ -61,6 +63,7 @@ func (h *StreamTranslatorHandler) ServeHTTP(w http.ResponseWriter, req *http.Req
 	// to the client.
 	websocketStreams, err := webSocketServerStreams(req, w, h.Options)
 	if err != nil {
+		metrics.IncStreamTranslatorRequest(req.Context(), strconv.Itoa(http.StatusInternalServerError))
```

Comment:
is this an internal error or a client error? seems like we'd want to treat this differently than spdy client construction / use errors, or status errors from the backend, or backend errors of unexpected types

Reply:
Yes. This is a client error. Incrementing `400 Bad Request`.